gentest/gentest-driver.ts

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
This script needs instructions on how to run it (especially given that it's a TS file that won't run with vanilla node):

- Minimum node version
- That you need to run NPM install
- That you need chromedriver (if you do - I already had it installed so can't be sure)
- The run command (`node --loader babel-register-esm gentest-driver.ts` ?). Perhaps this could be included in the package.json as a script. And then the instructions could just be `npm run gentest` or similar.

Reply:
Yeah in some of the later commits in this stack I update the readme for instructions on how to get this to work wrt to dependencies. We also have scripts in place: `yarn gentest` and `yarn gentest-validate` which abstracts that out that loader.

As for chromedriver, according to https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/#use-the-latest-version-of-selenium, it appears that selenium now handles those deps for you like playwright. I have not tested this yet though, I'll give it a try today